suse_openstack_cloud

```diff
@@ -268,6 +268,23 @@ plog_files 0 $crowbar_db_dump_file
 rm -f $crowbar_db_dump_file
 
 #############################################################
+section_header "MariaDB"
+
+find_and_plog_files  /var/log/mysql   -type f
+
+mariadb_variables_file=/tmp/mariadb_variables
+mariadb_status_file=/tmp/mariadb_status
+
+mysql -u monitoring -e 'SHOW VARIABLES;' > $mariadb_variables_file
```

Comment:
Please check https://github.com/crowbar/crowbar-openstack/pull/1277 , @jhesketh is replacing `monitoring` user with a new one